server/implementation-cdi/src/main/java/io/smallrye/graphql/cdi/tracing/TracingService.java

```diff
@@ -85,7 +83,7 @@ public void beforeDataFetch(Context context) {
         final Span span = scope.span();
 
         GraphQLContext graphQLContext = env.getContext();
-        graphQLContext.put(PARENT_SPAN_KEY, span);
+        graphQLContext.put(Span.class, span);
```

Comment:
Ok, that change how it worked before. (that I did not want to do in this PR) Maybe @jmartisk can look at this in another PR ?

Reply:
+1 for using the name only. This might potentially get us into trouble in multi-app environments if the `Span.class` is loaded by different class loaders, for example when someone deploys WildFly an application where the graphql jars are bundled in the app. Not that it's correct usage, but it's possible it will happen.
Yeah I can get that fixed if you prefer